### PR TITLE
Setting rel. error tol. for QAGS

### PR DESCRIPTION
**Pull request by ollele**
Implemented setting the relative error tolerance for QAGS used for screened bremsstrahlung. Edited example pi-file accordingly. Cleaned in plotspectrum.py

### CODE REVIEW

File: examples/Bremsstrahlung_screened/pi [hunk 1]
```diff
@@ -58,6 +58,7 @@ tools              = rad;
     Z = 10, 18;
     Z0 = 2, 2;
     n = 1e19, 1e19;
+    QAGS_EpsRel = 1e-3; #Rel error tolerance for QAGS, defualt is 1e-3
```
Comment (hoppe93) [inline examples/Bremsstrahlung_screened/pi]: For consistency, could you change the name of this setting ``qagstol``? (apparently I haven't documented this setting in ``RadiationModel (angdist)``)

File: src/Tools/Radiation/Models/Cone/Configure.cpp [hunk 2]
```diff
@@ -138,20 +138,27 @@ void Cone::ConfigureEmission(const string& emname, ConfigBlock *conf) {
             Z0[i] = Z0_vec[i];
             density[i] = n_vec[i];
         }
-        this->emission = new ConeBremsstrahlungScreenedEmission(this->parent->detector, this->parent->magfield, nspecies, Z, Z0, density);
+        slibreal_t qagsEpsRel = 1e-3;
+        if (conf->HasSetting("QAGS_EpsRel")){
```
Comment (hoppe93) [inline src/Tools/Radiation/Models/Cone/Configure.cpp]: Change name here please!

File: src/Tools/Radiation/Models/Cone/Configure.cpp [hunk 3]
```diff
@@ -138,20 +138,27 @@ void Cone::ConfigureEmission(const string& emname, ConfigBlock *conf) {
...
+        }
+        this->emission = new ConeBremsstrahlungScreenedEmission(this->parent->detector, this->parent->magfield, nspecies, Z, Z0, density, qagsEpsRel);
     }
-else if (emname == "synchrotron") {
-        this->emission = new ConeSynchrotronEmission(this->parent->detector, this->parent->magfield, this->parent->MeasuresPolarization());
-    } else if (emname == "unit") {
-        if (this->parent->MeasuresPolarization())
-            throw ConeUnitException("The 'Unit' emission model does not support polarization measurements.");
+    else if (emname == "synchrotron") {
+            this->emission = new ConeSynchrotronEmission(this->parent->detector, this->parent->magfield, this->parent->MeasuresPolarization());
```
Comment (hoppe93) [inline src/Tools/Radiation/Models/Cone/Configure.cpp]: The indentation from this line and onwards seems a bit off?

**Review by hoppe93 — CHANGES_REQUESTED** [review]
I have few comments which I'd prefer that you fix

**Comment by ollele** [comment]
Fixed setting name, and indentation

**Review by hoppe93 — APPROVED** [review]

**Pull request merged**